### ISSUE TITLE
Make "select day" option default to the current day's schedule

### ISSUE THREAD

**Issue by tektaxi** [Aspine/aspine#261]
If it's a weekday, we should have the "select day" option default to the schedule for that day. Or if it's a weekend, just have it show Monday's schedule (as it currently does, so no need to change that).

**Comment by psvenk** [comment]
@tektaxi If I'm not mistaken, this is the current behavior in #257.

**Comment by tektaxi** [comment]
I used the `date_override` variable in the console to test it out, and it didn't change how it behaves. Unless that variable doesn't actually affect that, I think it doesn't work like this right now.

**Comment by psvenk** [comment]
This is the case because the schedule does not automatically update all the time like the clock does. If you switch to another day and then go back to "select day", the schedule should update to the current day's.

While testing this, I actually found a bug where setting the date override to a Thursday and selecting Monday in the dropdown displayed a Thursday schedule, which I fixed in 951f4394b2. Thanks for opening this issue @tektaxi!
